tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala

```diff
@@ -234,4 +234,54 @@ class WorkspaceSymbolLspSuite extends BaseLspSuite("workspace-symbol") {
       )
     } yield ()
   }
+
+  test("excluded") {
+    cleanWorkspace()
+    for {
+      _ <- server.initialize(
+        """
+          |/metals.json
+          |{"a": {}}
+          |/a/src/main/scala/a/Before.scala
+          |package a
+          |object MyObjectSymbol
+          |""".stripMargin
+      )
+      _ <- server.didOpen("a/src/main/scala/a/Before.scala")
+      _ = assertNoDiff(
+        server.workspaceSymbol("Future"),
+        """|scala.concurrent.Future
+           |scala.concurrent.Future
+           |java.util.concurrent.Future
+           |scala.sys.process.ProcessImpl#Future
+           |java.util.concurrent.FutureTask
+           |scala.collection.parallel.FutureTasks
+           |java.io.ObjectStreamClass#EntryFuture
+           |java.util.concurrent.RunnableFuture
+           |java.util.concurrent.ExecutorCompletionService#QueueingFuture
+           |java.util.concurrent.ScheduledFuture
+           |java.util.concurrent.CompletableFuture
+           |java.util.concurrent.ScheduledThreadPoolExecutor#ScheduledFutureTask
+           |scala.collection.parallel.FutureThreadPoolTasks
+           |java.util.concurrent.RunnableScheduledFuture""".stripMargin
+      )
+      _ <- server.didChangeConfiguration(
+        """|{
+           |  "excluded-packages": [
+           |     "java.util/"
```

Comment:
user will type probably `java.util` without '/'

Reply:
Same as above, I'll remove the `/`.